plugins/varnish/varnish4_multigraph

```diff
@@ -0,0 +1,453 @@
+#!/usr/bin/python3 -tt
+# -*- coding: utf-8 -*-
+
+""" Munin plugin to monitor Varnish 4 status.
```

Comment:
Please convert this to the `perlpod` format. Users of the plugin gallery and `munin-get` will appreciate this :)